CHANGELOG.md

```diff
@@ -0,0 +1,48 @@
+# Changelog
+
+Because we use squash and merge, you should be able to see the changes by looking
+at the [commit log](commits/master). However, we have begun eeping track of breaking changes
+or optional rake tasks.
+
+### Welcome email refactoring/cleanup (#2362)
```

Comment:
We could also consider adding the merge date to make it easier for downstream developers to keep track of what's new. The only problem with that is then the change would need to be a separate commit made right before merging.

I would recommend that whoever makes the PR includes notes here as part of it.